how-to-audio-processing.md

```diff
@@ -0,0 +1,44 @@
+# Audio pre and post processing
+
+The WebRTC ADM in M71 enabled audio pre and post processing as described [here](https://bugs.chromium.org/p/webrtc/issues/detail?id=8665).  The same classes are available to managed code in WebRTC UWP via the WebRtcFactory class.  
+
+> Note that pre & post processing is not available per-stream.  The filters can be specified per-WebRtcFactory.  WebRtcFactory can be instantiated multiple times per application if separate filters are needed for separate streams or PeerConnections.
+
+> These methods are necessarily blocking, as they are in Chrome.  Handlers with long execution times could cause issues in the ADM.

```

Comment:
Also the setup allows specifying a "thread queue" for the audio callback. Recommendation should be used to get the callbacks out of the GUI dispatch thread which will likely cause issues by creating a thread queue for these callbacks.

Reply:
Added